test/visibility/except/deepUseSetup.chpl

```diff
@@ -21,7 +21,7 @@ module M2 {
 // a variable is not also defined elsewhere.  Non-transitive uses would prevent
 // the current behavior
 module M3 {
-  use M1;
+  public use M1;
```

Comment:
Since these tests exist to check use statements, it would be good to update their comments to refer to "public" uses rather than just uses.  For example, in this test I would update line 15 to say "and publicly uses M1" and line 21 to say "Private uses would prevent"